raiden-dapp/tests/unit/query-params.spec.ts

```diff
@@ -34,4 +35,19 @@ describe('query params', () => {
       expect(getAddress(undefined)).toBe('');
     });
   });
+
+  describe('paymentId', () => {
+    test('invalid param returns undefined', () => {
+      expect(getPaymentId('onehundred')).toBe(undefined);
+    });
+
+    test('valid string returns BigNumber', () => {
```

Comment:
Just to be correct:
If you agree to simply the `getPaymentId` function as suggested above you can make this more generic because it accepts everything that `bigNumberify` can handle, not only strings. But ofc we don't want to test the function of our dependencies, just make sure that formats we want to work are actually working. This could be 1-2 different string formats.

Reply:
Added hex encoded string test